library/ctr_drbg.c

```diff
@@ -180,7 +185,11 @@ static int block_cipher_df( unsigned char *output,
     for( i = 0; i < MBEDTLS_CTR_DRBG_KEYSIZE; i++ )
         key[i] = i;
 
-    mbedtls_aes_setkey_enc( &aes_ctx, key, MBEDTLS_CTR_DRBG_KEYBITS );
+    if ( 0 != mbedtls_aes_setkey_enc( &aes_ctx, key, MBEDTLS_CTR_DRBG_KEYBITS ) )
```

Comment:
Style nitpick: we don't do the `CONSTANT == VARIABLE` thing, let alone `CONSTANT == EXPRESSION`. For consistency, please stick to `if( mbedtls_aes_setkey_enc( … ) != 0 )` and the like.

Reply:
Done